services/frontend/pages/cool-hats.tsx

```diff
@@ -0,0 +1,95 @@
+import { Layout } from '@components/common'
+import { Container, Text } from '@components/ui'
+import { SEO } from '@components/common'
+import { GetServerSidePropsContext } from 'next'
+import { Product } from '@customTypes/product'
+import { ProductCard } from '@components/product'
+
+interface CoolHatsPageProps {
+  hatProducts: Product[]
+}
+
+export async function getServerSideProps(context: GetServerSidePropsContext) {
+  const baseUrl = process.env.NEXT_PUBLIC_FRONTEND_API_ROUTE
+    ? `${process.env.NEXT_PUBLIC_FRONTEND_API_ROUTE}/api`
+    : 'http://localhost:3000/api'
+
+  try {
+    // Fetch all products and filter for hat-related items
+    const allProducts: Product[] = await fetch(`${baseUrl}/products`)
+      .then((res) => res.json())
+      .catch(() => [])
+
+    // Filter for hat-related products (you can adjust this logic)
+    const hatProducts = allProducts.filter(product => 
+      product.name?.toLowerCase().includes('hat') ||
+      product.name?.toLowerCase().includes('cap') ||
+      product.name?.toLowerCase().includes('beanie') ||
+      product.description?.toLowerCase().includes('hat') ||
+      product.description?.toLowerCase().includes('headwear')
+    )
+
+    return {
+      props: {
+        hatProducts: hatProducts.slice(0, 12) // Limit to 12 products
+      }
+    }
+  } catch (error) {
+    console.error('Error fetching hat products:', error)
+    return {
+      props: {
+        hatProducts: []
+      }
+    }
+  }
+}
+
+export default function CoolHatsPage({ hatProducts }: CoolHatsPageProps) {
+  return (
+    <>
+      <SEO
+        title="Cool Hats - Storedog"
+        description="Discover our amazing collection of cool hats, caps, and headwear. Style meets comfort in our curated selection."
+      />
+      
+      <Container className="pt-8 pb-16">
+        {/* Hero Section */}
+        <div className="text-center mb-12">
+          <h1 className="text-4xl md:text-6xl font-bold text-gray-900 mb-4">
+            🧢 Cool Hats Collection
+          </h1>
+          <Text className="text-xl text-gray-600 max-w-2xl mx-auto">
```

Comment:
Let's replace all the `Text` components with a regular `p` tag, it'll fix the tailwind classes not being applied correctly
```suggestion
          <p className="text-xl text-gray-600 max-w-2xl mx-auto">
```